src/main/scala/com/gd/twitterstreamingtToDatalake/TransformTweets.scala

```diff
@@ -6,19 +6,18 @@ import twitter4j.Status
 object TransformTweets {
 
   def getText(englishTweets: DStream[Status]): DStream[String] = {
-    val text = englishTweets.flatMap(x => x.getText.split(" "))
+    val text = englishTweets.map(x => x.getText)
 
     text
   }
 
   def getHashTags(text: DStream[String]): DStream[String] = {
-    val hashTags = text.filter(_.startsWith("#"))
+    val hashTags = text.flatMap(_.split(" ")).filter(_.startsWith("#"))
 
     hashTags
   }
 
-  // This is the sample operation we are testing
```

Comment:
Could you clarify please, why have you decided to keep the "sample operation"? Does it relate to the given task?

Reply:
Yes.. You are right ! It was for personal debug.
Removed it !